jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/PdfStream.java

```diff
@@ -186,8 +205,17 @@ private void extractFilters () throws PdfException
             else {
                 /* Only other allowed value is a string */
                 Filter[] val = new Filter[1];
-                val[0] = new Filter 
-                    (((PdfSimpleObject) filter).getStringValue());
+                if(filter instanceof PdfSimpleObject) {
+                	val[0] = new Filter 
+                            (((PdfSimpleObject) filter).getStringValue());
+                } else {
+					System.out.println("we zijn er");
```

Comment:
Sorry I left some debug code (System.out.println("we zijn er");). This should be deleted

Reply:
That's the issue with bigger PRs and I let the dependency issue take my attention. I'll remove this and re-submit.